exercises/Stephen_Smith/ex02_oo_basics/include/Shapes.h

```diff
@@ -0,0 +1,218 @@
+#ifndef SHAPES_H_
+#define SHAPES_H_
+
+#include <string>
+#include <vector>
+#include <memory>
+#include <iostream>
+#include <math.h>  
+
+namespace{
+
+    const double PI=3.141592653589793238463;
```

Comment:
This can be `constexpr`. In the main codebase we would use the `M_PI` constant defined in `<cmath>`.